drivers/flash/flash_stm32_xspi.c

```diff
@@ -2163,7 +2164,10 @@ static int flash_stm32_xspi_init(const struct device *dev)
 
 #endif /* XSPIM */
 
-#if defined(DLYB_XSPI1) || defined(DLYB_XSPI2) || defined(DLYB_OCTOSPI1) || defined(DLYB_OCTOSPI2)
+#if !defined(CONFIG_SOC_SERIES_STM32H7RSX) && \
+	(defined(DLYB_XSPI1) || defined(DLYB_XSPI2) || \
+	 defined(DLYB_OCTOSPI1) || defined(DLYB_OCTOSPI2))
```

Comment:
This PR (https://github.com/zephyrproject-rtos/hal_stm32/pull/273/files#diff-c33769f659d0c0eb2dff17944f2dec9129a18177e106a47f1a553f852cc8d9f6) is removing definition of `DLYB_XSPI1` and `DLYB_XSPI1` for H7RS, so this line can be left untouched

Reply:
done